biodatasets/chemdner/chemdner.py

```diff
@@ -0,0 +1,392 @@
+# coding=utf-8
+# Copyright 2022 The HuggingFace Datasets Authors and Samuele Garda.

```

Comment:
sorry, this snuck in from a time when we were putting names in instead of using github to track contributions. can change this to the value in the template please? 
```
# Copyright 2022 The HuggingFace Datasets Authors and the current dataset script contributor.
```